scripts/deployment/haveno+external-tor+monerod-launcher.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Please rename this file to haveno-with-external-tor-monerod.sh for consistency with the other script.